CHANGELOG.md

```diff
@@ -40,6 +40,7 @@
 - Add support for building Android App Bundles (AABs)
 - Migrate `PatientsScreen` to Mobius
 - Migrated `SyncIndicatorController` to Mobius
+- Add extension for extracting intent data from `ActivityResult`
```

Comment:
I think these sort of changes can be skipped from the changelog since they don't really a major thing.

Reply:
Okay ✅